scripts/speaker_tasks/pathsfiles_to_manifest.py

```diff
@@ -120,11 +132,14 @@ def main(wav_path, text_path=None, rttm_path=None, uem_path=None, ctm_path=None,
         if ctm is not None:
             ctm = ctm.strip()
 
+        duration = None
+        if add_duration and sample_rate:
+            duration = librosa.get_duration(filename=audio_line, sr=sample_rate)
```

Comment:
Since, we support only inference of 16kHz files, you may pass a default value here.

Reply:
done